Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -3214,26 +3220,44 @@ def __init__(self):
         self.root_logger = logging.getLogger()
         self.prev_log_level = self.root_logger.getEffectiveLevel()
         self.root_logger.setLevel(logging.DEBUG)
+        self.org_handlers = list()
+        if self.root_logger.handlers:
+            self.org_handlers.extend(self.root_logger.handlers)
+            for h in self.org_handlers:
+                self.root_logger.removeHandler(h)            
         self.root_logger.addHandler(self.handler)
 
     def __del__(self):
         if self.handler:
             self.root_logger.setLevel(self.prev_log_level)
-            self.root_logger.removeHandler(self.handler)
+            self.root_logger.removeHandler(self.handler)            
             self.handler.flush()
             self.handler.close()
+        if self.org_handlers:
+            for h in self.org_handlers:
+                self.root_logger.addHandler(h)
         if self.http_client:
             self.http_client.HTTPConnection.debuglevel = 0
             if self.http_client_print:
                 setattr(self.http_client, 'print', self.http_client_print)
             else:
                 delattr(self.http_client, 'print')
+    
+    def log_start_debug(self):
+        """
+        Utility function to log start of debug mode logging
+        """
+        msg = "debug-mode started.\nhttp client print found: {}.\nEnv {}.".format(self.http_client_print is not None, os.environ)
+        if hasattr(demisto, 'params'):
+            msg = msg + "\nParams: {}.".format(demisto.params())
```

Comment:
this log will go through `_iter_sesistive_dict_obj` and secrets will be sanitized ?

Reply:
Yes. I just pushed also a unit test to test this out.